--- kiwisolver-1.4.4/setup.py.old	2022-08-26 14:10:48.785968917 +0200
+++ kiwisolver-1.4.4/setup.py	2022-08-26 14:11:21.736644810 +0200
@@ -46,4 +46,13 @@
 setup(
     ext_modules=ext_modules,
     cmdclass={"build_ext": CppyBuildExt},
+    name='kiwisolver',
+    version='1.4.5',
+    author='The Nucleic Development Team',
+    author_email='sccolbert@gmail.com',
+    url='https://github.com/nucleic/kiwi',
+    description='A fast implementation of the Cassowary constraint solver',
+    package_dir={"": "py"},
+    packages=["kiwisolver"],
+   install_requires=["typing_extensions;python_version<'3.8'"],
 )
